### PR TITLE
REFPLTB-1457 : Adding changes for wifi_getHalCapability

### PR DESCRIPTION
**Pull request by chilladamodarltts**
Signed-off-by: chilladamodarltts <chilla.damodar@ltts.com>

### CODE REVIEW

File: source/wifi/wifi_hal.c [hunk 1]
```diff
@@ -8719,9 +8720,174 @@ INT wifi_createVAP(wifi_radio_index_t index, wifi_vap_info_map_t *map)
...
+        printf("[wifi_hal dbg] : func[%s] line[%d] radio_index[%d] Invalid Country [%s]\n", __FUNCTION__, __LINE__, radioIndex, output_string);
+    }
+
+    rcap->numcountrySupported = 2;
+
+    /* csi */
+    rcap->csi.maxDevices = 8;
+    rcap->csi.soudingFrameSupported = TRUE;
+
+    snprintf(rcap->ifaceName, 64, "wifi%d_0", (radioIndex == 0) ? 2 : 0);
```
Comment (manigandanj) [inline source/wifi/wifi_hal.c:8811]: i think, radio interface name wlan0/wlan1.
with this code, radio interface name might be wifi0_0/wifi2_0.
please clarify

Comment (chilladamodarltts) [reply source/wifi/wifi_hal.c]: modified accordingly

File: source/wifi/wifi_hal.c [hunk 2]
```diff
@@ -8725,9 +8726,174 @@ INT wifi_createVAP(wifi_radio_index_t index, wifi_vap_info_map_t *map)
...
+        printf("[wifi_hal dbg] : func[%s] line[%d] radio_index[%d] Invalid Country [%s]\n", __FUNCTION__, __LINE__, radioIndex, output_string);
+    }
+
+    rcap->numcountrySupported = 2;
+
+    /* csi */
+    rcap->csi.maxDevices = 8;
+    rcap->csi.soudingFrameSupported = TRUE;
+
+    snprintf(rcap->ifaceName, 64, "wlan%d", (radioIndex == 0) ? 0 : 1);
```
Comment (manigandanj) [inline source/wifi/wifi_hal.c:8817]: To support triband in future, can we have simply radioIndex
snprintf(rcap->ifaceName, 64, "wlan%d", radioIndex);

Comment (chilladamodarltts) [reply source/wifi/wifi_hal.c]: done

**Review by manigandanj — APPROVED** [review]

**Pull request merged**